modules/Blockchain/Polygon/migrations/2_total_migration.js

```diff
@@ -161,6 +161,101 @@ module.exports = async (deployer, network, accounts) => {
             await hub.setContractAddress('Holding', holding.address);
             console.log('Updated contract in Hub contract map!');
         }
+        break;
+    case 'mainnet':
+        await deployer.deploy(Hub, { gas: 6000000, from: accounts[0] })
+            .then((result) => {
+                hub = result;
+            });
+
+        await deployer.deploy(Hub, { gas: 6000000, from: accounts[0] })
+            .then((result) => {
+                hub = result;
+            });
+        await hub.setContractAddress('Owner', accounts[0]);
+
+        profileStorage = await deployer.deploy(
+            ProfileStorage,
+            hub.address, { gas: 6000000, from: accounts[0] },
+        );
+        await hub.setContractAddress('ProfileStorage', profileStorage.address);
+
+        holdingStorage = await deployer.deploy(
+            HoldingStorage,
+            hub.address,
+            { gas: 6000000, from: accounts[0] },
+        );
+        await hub.setContractAddress('HoldingStorage', holdingStorage.address);
+
+        marketplaceStorage = await deployer.deploy(
+            MarketplaceStorage,
+            hub.address,
+            { gas: 6000000, from: accounts[0] },
+        );
+        await hub.setContractAddress('MarketplaceStorage', marketplaceStorage.address);
```

Comment:
I think that we don't need marketplace contract on Polygon since we are not going to use it